apps/microtvm/zephyr/template_project/src/mlperftiny/platform.cc

```diff
@@ -49,14 +55,8 @@ int8_t g_quant_zero = OUT_QUANT_ZERO;
 #endif
 size_t g_output_data_len = output_data_len;
 
-// WORKSPACE_SIZE is defined in python
-static uint8_t g_aot_memory[WORKSPACE_SIZE];
-tvm_workspace_t app_workspace;
-
-size_t TVMPlatformFormatMessage(char* out_buf, size_t out_buf_size_bytes, const char* fmt,
-                                va_list args) {
-  return vsnprintk(out_buf, out_buf_size_bytes, fmt, args);
-}
+static const struct device* g_microtvm_uart;
+static uint8_t uart_data[8];
```

Comment:
make `8` be a #define that gives a hint as to why the uart_data is 8 bytes

Reply:
that's no longer used, so I removed `uart_data`